internal/identity/user_data.go

```diff
@@ -111,11 +111,12 @@ var charmap = map[rune][]rune{
 }
 
 type UserData struct {
-	OK          bool
-	FirstNames  string
-	LastName    string
-	DateOfBirth date.Date
-	RetrievedAt time.Time
+	OK                   bool
+	FirstNames           string
+	LastName             string
+	DateOfBirth          date.Date
+	RetrievedAt          time.Time
+	InsufficientEvidence bool
```

Comment:
I was on the fence about including this but it feels like we will need a way to differentiate between a failed ID check and someone on the vouching route.

Reply:
I think it would be worth having an enum that replaces this and OK with `Passed/InsufficientEvidence/Failed` or something